src/main/resources/config/application-dev.yml

```diff
@@ -67,10 +71,11 @@ jhipster:
     authentication:
       jwt:
         # This token must be encoded using Base64 and be at least 256 bits long (you can type `openssl rand -base64 64` on your command line to generate a 512 bits one)
-        base64-secret: MDc2YTJkMGVmMTc3ZjQ3MjgwNjVkMDRhYmE1MThmZmNlZGMyMjgzNjgxMTMyZGY3MzlhOTA1Mjk1NzUwODgyOTZhODM1ZTYyZmI5YmJjZWVkNjVhZWM5NDgzMzU2OWZkYzUzNjVkYTNiMTIxZTlhMWRkYjJhMTg2ZWJmMzgxMjQ=
+        # base64-secret: MDc2YTJkMGVmMTc3ZjQ3MjgwNjVkMDRhYmE1MThmZmNlZGMyMjgzNjgxMTMyZGY3MzlhOTA1Mjk1NzUwODgyOTZhODM1ZTYyZmI5YmJjZWVkNjVhZWM5NDgzMzU2OWZkYzUzNjVkYTNiMTIxZTlhMWRkYjJhMTg2ZWJmMzgxMjQ=
```

Comment:
can you explain me why it is commented ?

Reply:
It's commented because this base64-secret key is not equal to the one used by default in eureka or consul when we generate an application with jhipster command. So, when we authenticate in Jhipster Control Center the generated token is not signed by the correct key and when we try to access to resources from microservices through the gateway, we have not a valid token.

Reply:
Ok, I think I understand:
- when using Eureka or Consul, the other microservice pull the configuration from Spring Cloud Config, so this value: https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/docker/config/localhost-config/application.yml.ejs#L10

So I think it's because the JHipster Control Center doesn't pull this configuration
Let's keep your change for now, so